user_docs/en/userGuide.t2t

```diff
@@ -2645,11 +2646,13 @@ To list add-ons only for specific channels, change the "Channel" filter selectio
 To search add-ons, use the "Search" text box.
 You can reach it by pressing ``shift+tab`` from the list of add-ons, or by pressing ``alt+s`` from anywhere in the Add-on Store interface.
 Type a keyword or two for the kind of add-on you're looking for, then ``tab`` back to the list of add-ons.
-The list will now show add-ons that match your search.
+Add-ons will be listed if the search text can be found in the display name, publisher or description.
+
+Add-ons can be filtered by display name, publisher and description.

```

Comment:
this is duplicated now

Reply:
@seanbudd I agree that it is, but I thought you asked me to bring back that language, which I had previously removed in a53058bff4c425cc0d7e0192435508d09922fe84.
Evidently I misunderstood your intent. I'll pull the filter line again.